### ISSUE TITLE
Develop a GUI for file viewing

### ISSUE THREAD

**Issue by wace-vault** [wace-vault/collection#2]
In extension of #1, some files aren't able to render properly or render at all when viewed in GitHub, which could also be addressed.

**Comment by denosawr** [comment]
Should be possible to render .docx using the MS Office viewer, at ([source](https://stackoverflow.com/questions/27957766/how-do-i-render-a-word-document-doc-docx-in-the-browser-using-javascript)):

```html
<iframe
  src='https://view.officeapps.live.com/op/embed.aspx?src=http%3A%2F%2Fieee802%2Eorg%3A80%2Fsecmail%2FdocIZSEwEqHFr%2Edoc'
  width='100%'
  height='100%'
  frameborder='0'
>
  This is an embedded 

  <a target='_blank' href='http://office.com'>
    Microsoft Office
  </a>

  document, powered by

  <a target='_blank' href='http://office.com/webapps'>
    Office Online
  </a>.
</iframe>
```

**Comment by iJynx** [comment]
Github pages could be a potential solution however i've never worked with it and i don't know how capable it is to embed files

**Comment by wacedungeoner** [comment]
> Github pages could be a potential solution however i've never worked with it and i don't know how capable it is to embed files

@denosawr's solution should theoretically work independent of what we use to host the website.

**Comment by wace-vault** [comment]
Closed with #13.
